Remove background circle from the Icons and Logos used by DOSBox Staging

ThomasEricB opened the pull request:
Continuation of https://github.com/dosbox-staging/dosbox-staging/pull/2229, but better made. 

The idea is: remove the circle to make the executable icons cleaner (nothing to do with the favcon from the website nor the website). Basically this:

![firefox_ZGh9UimAZt](https://user-images.githubusercontent.com/9662095/213121784-b2ed0702-fef8-49b0-b6bc-a860722c19f4.png)

@kcgen if you could "defer review and acceptance to @johnnovak " again that would be great

kcgen commented:
> @kcgen if you could "defer review and acceptance to @johnnovak " again that would be great

:+1: ; added @johnnovak as reviewer.

ThomasEricB commented:
To clarify: this latest commit is just a supplementary one and my last one, I had to do another one since I didn't know that you had to type "git rm" to actually remove a file. Ended up using this opportunity to edit the icons.md to add new information - Information that I would've included on the first git commit description, if I'd knew how to add it the first time around.

johnnovak commented:
Here's a before/after comparison on macOS. I personally prefer the existing icon; it stands out better in the installer and the App Switcher, and generally it blends in more with the other app icons that also have a border. Perhaps we could even improve the macOS icon to convert the circle into a rounded rectangle, so it would follow macOS icon standards more closely. But I think the border should stay, on macOS at least. What do other macOS users think? @kcgen , @kklobe , @Burrito78 ?

I haven't tested it on Windows yet, but before I do that, could you please post screenshots demonstrating how this change improved things on your end? Note I cannot test this on Linux myself.

# macOS – existing icons

## Installer

<img width="654" alt="image" src="https://user-images.githubusercontent.com/698770/213347045-403f1f6d-eb38-468d-a67d-53a5a142c9c2.png">

## Applications

<img width="665" alt="image" src="https://user-images.githubusercontent.com/698770/213347140-de04286e-4419-48cb-8524-4ed3438f632d.png">

## App Switcher

![image](https://user-images.githubusercontent.com/698770/213347313-9e909426-4c7a-48a1-a03d-97ecbd83ad5d.png)



# macOS – new icons

## Installer

<img width="654" alt="image" src="https://user-images.githubusercontent.com/698770/213346368-93c11eed-331f-4d91-a3db-bbebbc5d4edf.png">

## Applications

<img width="986" alt="image" src="https://user-images.githubusercontent.com/698770/213346768-f2dc79e8-3f90-4dca-bdbf-6801e27346fa.png">

## App Switcher

![image](https://user-images.githubusercontent.com/698770/213346603-4810b745-20ba-4248-9905-cdaa4ed3d035.png)

ThomasEricB commented:
> Here's a before/after comparison on macOS. I personally prefer the existing icon; it stands out better in the installer and the App Switcher, and generally it blends in more with the other app icons that also have a border. Also, the new icon is too big. Perhaps we could even improve the macOS icon to convert the circle into a rounded rectangle, so it would follow macOS icon standards more closely. But I think the border should stay, on macOS at least. What do other macOS users think? @kcgen , @kklobe , @Burrito78 ?

I agree with you about that, it does look weird in MAC. Looking in the screenshots you provided, what would you think about my testing something like the Firefox logo? Because the current one does look out of the place as well.

johnnovak commented:
> Looking in the screenshots you provided, what would you think it being something like the Firefox logo? Because the current one does look out of the place as well.

It does, doesn't it? If you could make a variant for macOS that is similar to the rest of the app logos, e.g. the Firefox one, that would be cool. I approve 👍🏻 😄

kcgen commented:
I agree @johnnovak ; seeing the side-by-side comparisons, I can see how the rounded-square border would make it look more consistent with the other icons - even more so than the circle border.

So at least on macOS:

   rounded-square > circle > bare

On Linux icons typically don't have a border:

![2023-01-18_19-35](https://user-images.githubusercontent.com/1557255/213349576-6896d3da-4f3c-4475-9f1b-cd82eac77a1c.png)

So the bare icon would look better there (so long as the background isn't matching white).

Would it make sense to offer these along-side the existing icons, so user can pick?

johnnovak commented:
> Would it make sense to offer these along-side the existing icons, so user can pick (in addition to the macOS-styled version, as well)?

How would that work, the icon selection? On macOS at least, users are happy with whatever they get; I don't think any Mac app has the option to switch the app icon. The situation on Windows is similar.

kcgen commented:
On most Linux window managers, you can select any old icon or image file to represent a launch shortcut (if it supports the format).

ThomasEricB commented:
I made these two on photoshop:

![Photoshop_Li5WG3QNHp](https://user-images.githubusercontent.com/9662095/213359855-6faa1307-bcc9-4547-9d6f-e74f5b2b8f80.png)
![Photoshop_ryPeBTHbbd](https://user-images.githubusercontent.com/9662095/213359859-25dae734-9f89-4733-aec8-ce6eb54d51cc.png)

Tell me what y'all think

ThomasEricB commented:
@johnnovak suggested that I made variations for the new Mac Icon, just so we can vote and discuss which ones we like, what we have to change, etc. So here are the ones I made:

### Normal colors:

• Title: **"Simple Mac"** ; If you want this one vote with 👀 below. ![Eyes](https://github.githubassets.com/images/icons/emoji/unicode/1f440.png)

![Simple Mac](https://user-images.githubusercontent.com/9662095/213372828-d576799c-1878-4325-8bff-9e8193b071fe.png)

• Title **"Glossy Ver. 1"**; If you want this one vote with 🎉below. ![tada](https://github.githubassets.com/images/icons/emoji/unicode/1f389.png)

![Glossy 1 Mac](https://user-images.githubusercontent.com/9662095/213373740-744ce9fc-94c2-49ca-8194-f0d1eff6268b.png)

• Title **"Glossy Ver. 2"**; If you want this one vote with 🚀 below. ![rocket](https://github.githubassets.com/images/icons/emoji/unicode/1f680.png)

![Glossy 2 Mac](https://user-images.githubusercontent.com/9662095/213374969-a5e36239-dbc6-4f5e-916b-6fad0677e6b2.png)

### Desaturated versions: 

• Title **"Simple Mac Desaturated"**; If you want this one vote with 😄 below. ![smile](https://github.githubassets.com/images/icons/emoji/unicode/1f604.png)

![Simple Mac desaturated](https://user-images.githubusercontent.com/9662095/213376050-3645e19a-8cb8-4430-9d19-10087c2a51f3.png)

• Title **"Glossy Ver. 1 Desaturated"**; If you want this one vote with ❤️ below. ![heart](https://github.githubassets.com/images/icons/emoji/unicode/2764.png)

![Glossy 1 Mac desaturated](https://user-images.githubusercontent.com/9662095/213377406-bf8850dd-f723-4fcf-9c29-68b0b0106364.png)

• Title **"Glossy Ver. 2 Desaturated"**; If you want this one vote with 👍 below (I ran out of emojis!). ![up](https://github.githubassets.com/images/icons/emoji/unicode/1f44d.png)

![Glossy 2 Mac desaturated](https://user-images.githubusercontent.com/9662095/213377881-6589e3da-6aab-4905-9e1d-73b548e0d374.png)

• **And finally, if you dislike them all or you want changes to be made in any of them vote with thumbs down.**

I would like to ask frequent contributors for help deciding this change! So I would like to get the input from @kcgen @Wengier @kklobe @rsribeiro @rderooy @FeralChild64 @shermp @dreamer for help here.

Grounded0 commented:
You can use the circleless for Linux and Windows and the box for macOS since its recommended by macOS Human Interface Guidelines for applications:

"In macOS, app icons share a common set of visual attributes, including a rounded-rectangle shape, front-facing perspective, level position, and uniform drop shadow. Rooted in the macOS design language, these attributes showcase the lifelike rendering style people expect in macOS while presenting a harmonious user experience."

https://developer.apple.com/design/human-interface-guidelines/foundations/app-icons/

👍

ThomasEricB commented:
> You can use circleless icon for Linux and Windows and the rounded box for macOS 

That's the idea, at least for now.

Burrito78 commented:
What @Grounded0 said basically. Following Apples macOS Human Interface Guidelines as close as possible is the first step in producing a professional looking icon, imho.

Templates are here to make your life easier:
https://developer.apple.com/design/human-interface-guidelines/guidelines/overview/

For comparisons it is easier to include smaller side-by-side overviews of the different versions.
Also include @johnnovak's screenshots above to make comparisons to existing and popular app icons easier.

Thats a good start, rounded corners here we come!!

Grounded0 commented:
I'm casting my vote for "Simple Mac" option.

johnnovak commented:
I like the first "simple" icon too without the glossy overlays. I would like to see a few variations where the padding is increased; now the icon is a bit "stuck" to the encompassing rounded rectangle. 

Could you post a few variations in a row in a smaller size? Current one, padding +10%, padding +20%, padding +30%, something like that.

Also, as it was suggested by others, it would help if you put it next to some of the application icon screenshots I sent you on Discord, so we can see how it looks in relation to common app icons.

Burrito78 commented:
The Firefox icon is a good comparison of how our new icon should look. It was recently updated in FF 106.

The discussion is found here, it's an interesting read:
https://bugzilla.mozilla.org/show_bug.cgi?id=1648386

Also this thread (how to create different sizes for different icon sizes/DPI):
https://bugzilla.mozilla.org/show_bug.cgi?id=1784524

ThomasEricB commented:
Okay, since @johnnovak asked, I'm also posting a version with a smaller logo on gray and a version where it resembles the DOSBox Terminal. Plus some new icons that I made trying to resemble the look of the terminal. See:

• Title: **"John Novak's DOSBox"** ; If you want this one vote with 👀 below. ![Eyes](https://github.githubassets.com/images/icons/emoji/unicode/1f440.png)

![John Novak's DOSBox](https://user-images.githubusercontent.com/9662095/213421857-ead33f4e-2009-48ad-8ed3-705674d3df30.png)

• Title **"OG's DOSBox Ver. 1"**; If you want this one vote with 🎉below. ![tada](https://github.githubassets.com/images/icons/emoji/unicode/1f389.png)

![OG's DOSBox 1](https://user-images.githubusercontent.com/9662095/213424408-7f6d6097-bb29-47ac-a4e0-2256992eb134.png)

• Title **"OG's DOSBox Ver. 2"**; If you want this one vote with 🚀 below. ![rocket](https://github.githubassets.com/images/icons/emoji/unicode/1f680.png)

![OG's DOSBox 2](https://user-images.githubusercontent.com/9662095/213424608-cfad9cbe-774d-4d5a-a1db-6bdb63dff70a.png)

johnnovak commented:
To me the "John Novak's DOSBox" version, just with dark grey background like on the other with the "Z:>" prompt would be perfect, I'd sign off on that 🚀 That's my final vote.

I don't the ones with the "Z:>" prompt, especially not the last one where the DOSBox logo is offset horizontally.

rderooy commented:
I like the extra space around the logo for the mac. Before it looked too cramped.

I also prefer the black background, but don't like the prompt.

Here is how things look today on Gnome.
![image](https://user-images.githubusercontent.com/17701098/213426162-f42318fe-fa6c-44b8-ada1-9798c8e93fd3.png)

johnnovak commented:
On Windows the legibility of the new icon is better as you can see here, but it blends too much into the grey background. So maybe you could give it an outline or something. I think it's an improvement over the old icon which apparently was downscaled using nearest-neighbour interpolation... ugh!

![image](https://user-images.githubusercontent.com/698770/213428463-c52c0078-141e-47d9-ae51-0dbb7a9cb560.png)

By the way, somehow only the icon of the log window got changed:

![image](https://user-images.githubusercontent.com/698770/213428550-8e87e9a3-bdd9-4ccc-8fe5-2865ee368f79.png)

Looks good in the file explorer too, but again, it blends into the background, it needs an outline I think.

![image](https://user-images.githubusercontent.com/698770/213429455-2531e6cc-3a33-4e79-8a9a-67766795d16c.png)

Grounded0 commented:
It needs a black outline for sure without a circle or otherwise there's blending issues with white-on-white. As for macOS logo nothing too crazy or out of the line or people are going to crazy about it. The best icon is the one that annoys least amount of people.

Grounded0 commented:
I like the black background on the macOS icon because it reminds me of DOS command line.

ThomasEricB commented:
> It needs a black outline for sure without a circle or otherwise there's blending issues with white-on-white. 

I added one in today's commit [71c0816](https://github.com/dosbox-staging/dosbox-staging/pull/2232/commits/71c08167def7928e40bec6a0f3bc49eb5a5563bd).

ThomasEricB commented:
Cool feature I just added... Now even the windows .ico is legible even at the very low resolutions like the original dosbox (NOT DOSBox Staging!) icons  (before it was a blurry mess): 
![icofx3_QlIrXmH9rv](https://user-images.githubusercontent.com/9662095/213526038-91b7d38b-7016-4134-9aaf-c5512cffe289.png)

ThomasEricB commented:
@johnnovak You should be all set now to test and review. Since yesterday I have:

• Changed the MAC Icon to one you suggested **(.icns)**
• Added an black outline and improved readability to the Windows icons **(.ico)**
• Added an black outline to the Linux icons **(.svgs)**

johnnovak commented:
> @johnnovak You should be all set now to test and review. Since yesterday I have:
> 
> • Changed the MAC Icon to one you suggested **(.icns)**
> • Added an black outline and improved readability to the Windows icons **(.ico)**
> • Added an black outline to the Linux icons **(.svgs)**

That sound like a plan, but needs more work. This is what I'm getting with the latest build.

You'll need to look into a bit more how the icons are applied to the exe/installer/etc per platform. If you're stuck, someone here can help 😄 

<img width="654" alt="image" src="https://user-images.githubusercontent.com/698770/213584391-249a2b89-2c0d-4b32-8e95-922d0b9987d7.png">

ThomasEricB commented:
> You'll need to look into a bit more how the icons are applied to the exe/installer/etc per platform. If you're stuck, someone here can help 😄

The issue you just revealed is this: the MAC version uses the .svg file instead of it's .icns file I made. It's the same deal you encountered here https://github.com/dosbox-staging/dosbox-staging/pull/2232#issuecomment-1396821409 I believe. This is going to be an issue as that If I add the squared circle on the SVG file, it will add to all platforms. 

@kcgen We need to make sure the Mac Icon (.icns) goes to Mac, the Windows Icon (.ico) goes to Windows and the svg files goes to linux. Right now Mac is still using the .svg for instance. Do you know how I can edit/ separate the platforms here?

Grounded0 commented:
Looking at my Dock the whole macOS HIG icon rule isn't strictly followed so we could go with the same icon for all platforms:

<img width="107" alt="Screenshot 2023-01-20 at 3 15 58" src="https://user-images.githubusercontent.com/48192273/213597373-ad1048c2-ac5c-4b79-9ac9-993a173c7061.png">

johnnovak commented:
> Looking at my Dock the whole macOS HIG rounded-rectangle shape icon rule isn't strictly followed so we could go with the same icon for all platforms

I think most of us Mac users always knew it isn't strictly followed, but the new icon with the rounded rectangle is an improvement as it looks better and aligns more closely with those guidelines, so IMO so we should switch to it on the Mac.

Grounded0 commented:
Yeah but we need to change the code so different icons can be delivered on different platforms. Shouldn't be too hard but its extra work related to this.

johnnovak commented:
> Yeah but we need to change the code so different icons can be delivered on different platforms. Shouldn't be too hard but its extra work related to this.

I personally never thought the old icons needed changing, but I like improvements @ThomasEricB has made. So I'm like 50/50.

I don't think the required changes would take more than an hour or two to do, I'll let @kcgen to decide.

kcgen commented:
The original icon was solid and this is a nice refinement, especially on more icon-centric platforms like Steam where icons look good without backgrounds.

And if @johnnovak thinks it'll be a good fit for the website too - then even better :1st_place_medal: !

Happy to let everyone else decide when it's ready to merged - the oulined version is going to look great on Linux too (given it expands the size bringing it on-par with other icons).

johnnovak commented:
> And if @johnnovak thinks it'll be a good fit for the website too - then even better 🥇 !

I wouldn't do any changes to the website; you might have noticed that I've already created a "circle-less" version of the icon for the top header bar. But I like to have the circle on the front page and on the about page around the big icons in the content section below; that brings some variety into the design. It's not just like we're repeating the exact same icon from the top bar, only in a bigger version (I tried that back before the site went online, and I didn't like the results...)

<img width="1385" alt="image" src="https://user-images.githubusercontent.com/698770/213622753-db59b4c2-d95f-4a21-b152-c1c7585a16ec.png">
<img width="1378" alt="image" src="https://user-images.githubusercontent.com/698770/213622437-3cf3f44d-945d-4696-810b-80b435a50fdb.png">

johnnovak commented:
@ThomasEricB Hey mate, just a friendly question — are you going to finish the job you started sometime?

ThomasEricB commented:
> @ThomasEricB Hey mate, just a friendly question — are you going to finish the job you started sometime?

This weekend I'll take a look on it, I swear!

johnnovak commented:
> > @ThomasEricB Hey mate, just a friendly question — are you going to finish the job you started sometime?
> 
> This weekend I'll take a look on it, I swear!

Okay man, there's no real urgency here, when you feel like it! Just from personal experience as elapsed time since the start of a task increases, the likelihood of ever finishing it decreases 😄 So it was just a friendly reminder with that in mind, but whenever you're free to work on this! 😁

kcgen commented:
Lets put this on the back-burner for now because it doesn't need reviewers and isn't ready for merge.
@ThomasEricB , when you've got more changes in the queue, please re-open! :+1:

Grounded0 commented:
I'm going to announce a 200 USD bounty for anyone who finishes this off prior to DOSBox Staging 0.81 release. Let's see what happens.

johnnovak commented:
So to sum up, we were saying we like the below icon for macOS, but without the prompt and of course the logo centered. Basically, we're following the rounded rectangle style on macOS.

Then for Windows and Linux we'll go with a black outline, otherwise the logo would blend into white / light backgrounds. Something that looks good, I'll sort it out.

Does that sound about right @Grounded0?


<img width="741" alt="image" src="https://github.com/dosbox-staging/dosbox-staging/assets/698770/2a3eafed-6789-4666-8317-a4dc48975078">

Grounded0 commented:
Yeah that sounds about right.

ThomasEricB commented:
@Grounded0 @johnnovak This is already mostly finished, we just need to fix the mac script. See the current files in the commits

Grounded0 commented:
Since i'm paying @johnnovak for this i'm not allowing him to review his own work 😁

Grounded0 commented:
0.81 release notes:

• Modernised application icons

The new icons are compliant with modern OS interface guidelines and improve icon legibility in small icon sizes. A custom icon has been added for macOS that conforms to the Human Interface Guidelines with a rectangle with rounded corners design.

Grounded0 commented:
When doing the rounded rectangle try to make our logo bigger inside the icon to improve icon legibility at small sizes. Nothing ridiculous of course. This looks okayish on a screenshot but looks like garbage on a 220 dpi HiDPI screen:

<img width="57" alt="Screenshot 2023-11-26 at 7 44 53" src="https://github.com/dosbox-staging/dosbox-staging/assets/48192273/8671aaa9-dfd8-447b-bb5c-2cc31c01e8c1">

johnnovak commented:
> When doing the rounded rectangle try to make our logo bigger inside the rounded rectangle to improve icon legibility at small sizes.

Of course, I'm gonna finish my in-flight viewport PR first, then I'll do the icon stuff as relaxation before tackling the video aspect resolution fix which is my final technical ticket.

The macOS icon is gonna be similar to the Chrome and Firefox logos on the below screenshot. 

<img width="1307" alt="image" src="https://github.com/dosbox-staging/dosbox-staging/assets/698770/36ba4b7f-9de2-4fdd-89d5-f8daded66df4">

Then I'll probably make the outline a bit thicker for the Linux & Windows icons, but yeah it needs testing at small icon sizes and fine-tuning so the pixel grid aligns just the right way when downscaling. E.g.:

<img width="544" alt="image" src="https://github.com/dosbox-staging/dosbox-staging/assets/698770/e90afbdf-324f-427a-bb2a-cc09d0276dd7">

Grounded0 commented:
Really hoping that RC is out by December or i'm starting to lose my hope for this project to be able to ship.

johnnovak commented:
> Really hoping that the RC is out by December or i'm starting to lose my hope for this project to be able to ship. I have over 3,000 USD tied up to this release so it can't become Duke Nukem Forever.

It will be out man 😎 Can you say with a straight face you lost hope when you look at the list of massive improvements in the release notes? 😄

Grounded0 commented:
Shipping is shipping and having something stashed away in `main` for the 10% cool guys is an another thing. This happened with DOSBox so i'm always wary is it going to happen again.

johnnovak commented:
> Shipping is shipping and having something stashed away in `main` for the 10% cool guys is an another thing. This happened with DOSBox so i'm always wary is it going to happen again.

Well... we're gonna aim for for 1 max 2 releases per year going forward. That's nice leisurely pace. Waiting 10 years for a new release vs waiting a few months is not the same thing 😄 It will be out in Dec, as I said. It's on me 90%.

johnnovak commented:
I'll use this PR to settle on the final look, then raise a new PR for the final review.

@Grounded0 @Burrito78 How are we liking this? I've muted the colours a _little bit_ too because the yellow and the green looked quite radioactive 😅 That has been bothering me for a while.

![image](https://github.com/dosbox-staging/dosbox-staging/assets/698770/b2f60616-4754-4513-897b-b17b7382d2e3)

Grounded0 commented:
![image](https://github.com/dosbox-staging/dosbox-staging/assets/48192273/73c1ab2c-0cff-4b19-b418-7b802a297f08)

Burrito78 commented:
> @Grounded0 @Burrito78 How are we liking this? I've muted the colours a _little bit_ too because the yellow and the green looked quite radioactive 😅 That has been bothering me for a while now.
> 
> Btw, I'll use this PR to settle on the final look, then raise a new PR for the final review.
> 
> ![image](https://private-user-images.githubusercontent.com/698770/285783738-b2f60616-4754-4513-897b-b17b7382d2e3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDEwODA0MTksIm5iZiI6MTcwMTA4MDExOSwicGF0aCI6Ii82OTg3NzAvMjg1NzgzNzM4LWIyZjYwNjE2LTQ3NTQtNDUxMy04OTdiLWIxN2I3MzgyZDJlMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMTI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTEyN1QxMDE1MTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xYmYyZTE0ZGRlMDRjNTk1OWJlN2Y4YmI3ZTFiNDc4ZjZjMTU5MzgxZGM0ZjY5ODRkMDQ0N2NhMmJjYzQzODZkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.tI4WpkvOnRPvTTa1J3qbV0S4k7ujrxY7n0d9F2wQHF8)

Love it!

johnnovak commented:
![image](https://github.com/dosbox-staging/dosbox-staging/assets/698770/2624462a-dbe3-47fa-97f2-0d485370812d)

MasterO2 commented:
> > Shipping is shipping and having something stashed away in `main` for the 10% cool guys is an another thing. This happened with DOSBox so i'm always wary is it going to happen again.
> 
> Well... we're gonna aim for 1 max 2 releases per year going forward. That's nice leisurely pace. Waiting 10 years for a new release vs waiting a few months is not the same thing 😄 It will be out in Dec, as I said. It's on me 90%.

On that note, December 1st is **this Friday**.

kklobe commented:
> > > Shipping is shipping and having something stashed away in `main` for the 10% cool guys is an another thing. This happened with DOSBox so i'm always wary is it going to happen again.
> > 
> > 
> > Well... we're gonna aim for 1 max 2 releases per year going forward. That's nice leisurely pace. Waiting 10 years for a new release vs waiting a few months is not the same thing 😄 It will be out in Dec, as I said. It's on me 90%.
> 
> On that note, December 1st is **this Friday**.

And December 31st 11:59PM is still a whopping 800 hours away!

ThomasEricB commented:
> Btw, I'll use this PR to settle on the final look, then raise a new PR for the final review.

Are you going to use my Windows/Linux icons? They are already done

johnnovak commented:
> > Btw, I'll use this PR to settle on the final look, then raise a new PR for the final review.
> 
> Are you @johnnovak going to use my Windows/Linux icons? They are already done

I'm starting from scratch because I've changed the colours a bit. Probably I'll make the border a little bit thicker too. I'll most likely need to tinker around with the pixel mapping for the low-res icons (16x16, 24x2r, etc.) so they look as good as they can. So yeah, starting from a clean slate.

johnnovak commented:
> > > > Shipping is shipping and having something stashed away in `main` for the 10% cool guys is an another thing. This happened with DOSBox so i'm always wary is it going to happen again.
> > > 
> > > 
> > > Well... we're gonna aim for 1 max 2 releases per year going forward. That's nice leisurely pace. Waiting 10 years for a new release vs waiting a few months is not the same thing 😄 It will be out in Dec, as I said. It's on me 90%.
> > 
> > 
> > On that note, December 1st is **this Friday**.
> 
> And December 31st 11:59PM is still a whopping 800 hours away!

I was thinking the RC would be a nice Christmas present for all DOS enthusiasts. That would be cool and seems doable.

johnnovak commented:
Continued in https://github.com/dosbox-staging/dosbox-staging/pull/3157
